app/views/users/show.html.erb

```diff
@@ -0,0 +1,98 @@
+
+<div class="container-fluid w-100 px-5 mb-5">
+  <h3 class="text-center ml-5 mt-4 text-white">
+    Event List <strong> <%= @user.name %> </strong> Created
+    <span class="btn btn-success ml-5 p-2">
+      <%= link_to '+ Create New Event', new_event_path, class: "text-white p-2 nav-link link-color-danger" %>
+    </span>
+
+  </h3>
+  <div class="row mb-5 p-0">
+    <div class="col-xl-12">
+      <table class="table">
+          <thead class="thead-dark">
+              <tr>
+                <th>Status</th>
+                <th>Description</th>
+                <th>Creator</th>
+                <th>Date</th>
+                <th>Location</th>
+                <th>Event Page</th>
+                <th>Delete</th>
+              </tr>
+          </thead>
+          <tbody>
+            <% @events.order(date: :desc).each do |event| %>
+              <tr class="table-primary align-middle">
+                <td class="align-middle">
+                  <%= badge_selector(event) %>
+                </td>
+                <td class="align-middle">
+                  <%= event.description %>
+                  <div class="badge badge-pill badge-info">
+                    <%= event.attendees.count %>
+                  </div> 
+                </td>
+                <td class="align-middle"><%= event.creator.name %></td>
+                <td class="align-middle"><%= event.date.strftime('%Y-%m-%d %H:%M %Z') %></td>
+                <td class="align-middle"><%= event.location %></td>
+                <td class="align-middle"><%= link_to 'Details', event_path(event.id), class: "btn btn-success btn-sm px-5" %></td>
+                <td class="align-middle"><%= delete_upcoming_event_button(event) %></td>
+              </tr>
+              <% end %>
+          </tbody>
+      </table>
+
+    </div>
+  </div>
+  <h3 class="text-center my-4 text-white">Event List <strong> <%= @user.name %> </strong> Invited </h3>
+  <div class="row">
+    <div class="col-xl-6">
+      <h5 class="text-center my-1 text-white">Upcoming</h5>
+      <table class="table">
+          <thead class="thead-dark">
+              <tr>
+                  <th>Description</th>
+                  <th>Location</th>
+                  <th>Date</th>
+                  <th>Event Page</th>
+              </tr>
+          </thead>
+          <tbody>
+            <% @upcoming_events.each do |event| %>
+              <tr class="table-primary">
```

Comment:
- [ ] **[OPTIONAL]** Your application uses this template (single event template) alot. To dry our views we can use [partials](https://guides.rubyonrails.org/layouts_and_rendering.html#using-partials) along with [collection rendering](https://guides.rubyonrails.org/layouts_and_rendering.html#rendering-collections).